app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -115,7 +115,8 @@ fun UnioApp(imageRepository: ImageRepositoryFirebaseStorage) {
         EventScreen(
             navigationAction = navigationActions,
             eventViewModel = eventViewModel,
-            userViewModel = userViewModel)
+            userViewModel = userViewModel,
+            mapViewModel)
```

Comment:
Keep the way you input the parameters consistent, e.g. `mapViewModel = mapViewModel`